ledger/eval/eval.go

```diff
@@ -645,6 +647,24 @@ func (cs *roundCowState) Move(from basics.Address, to basics.Address, amt basics
 	return nil
 }
 
+// autoHeartbeat compares `before` and `after`, returning a new AccountData
+// based on `after` but with an updated `LastHeartbeat` if `after` shows enough
+// balance increase to risk a false positive suspension for absenteeism.
+func (cs *roundCowState) autoHeartbeat(before, after ledgercore.AccountData) ledgercore.AccountData {
```

Comment:
My question earlier about falling just under the 2x threshold stands, but not sure its worth trying to cover every case.

Reply:
I'm going to resolve this convo, but we can further discuss at your other comment if you want.